core/src/utils/animation/animation-utils.ts

```diff
@@ -77,7 +94,7 @@ export const createKeyframeStylesheet = (keyframeName: string, keyframeRules: st
 
   const stylesheet = (element.ownerDocument || document).createElement('style');
   stylesheet.id = keyframeName;
-  stylesheet.textContent = `@keyframes ${keyframeName} { ${keyframeRules} } @keyframes ${keyframeName}-alt { ${keyframeRules} }`;
+  stylesheet.textContent = `@${keyframePrefix}keyframes ${keyframeName} { ${keyframeRules} } @${keyframePrefix}keyframes ${keyframeName}-alt { ${keyframeRules} }`;
```

Comment:
Same as previous comment. We can just grab the value of the variable now.

```suggestion
  stylesheet.textContent = `@${animationPrefix}keyframes ${keyframeName} { ${keyframeRules} } @${animationPrefix}keyframes ${keyframeName}-alt { ${keyframeRules} }`;
```

Reply:
See my comments above